### PR TITLE
beta3 and Update READEME.md

### PR DESCRIPTION
**Pull request by hasithajayasundara**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +19,21 @@ Using Ballerina SonarQube connector you can easily get important code quality me
 
 The following sections provide you with information on how to use the Ballerina SonarQube connector.
 
-- [Getting started](#getting-started)
-- [Working with sonarqube connector actions](#working-with-sonarqube-connector-actions)
+| Ballerina Language Version                   | Connector Version           | API Version
+| ---------------------------------------------|:--------------------------:| :--------------:
+| 0.970.0-beta3                                | 0.8.7                      | 6.7.2
```
Comment (Biruntha) [inline README.md]: Remove the Connector Version from here

File: README.md [hunk 2]
```diff
@@ -15,17 +19,21 @@ Using Ballerina SonarQube connector you can easily get important code quality me
...
 
 ## Getting started
 
-1. Clone Ballerina (https://github.com/ballerina-lang/ballerina) repository and run the Maven command ``mvn clean install`` from the ``ballerina`` root directory.
-2. Extract the Ballerina distribution created at `distribution/zip/ballerina/target/ballerina-<version>-SNAPSHOT.zip`  and set the PATH environment variable to the bin directory .
+1.  Refer https://stage.ballerina.io/learn/getting-started/ to download and install Ballerina.
```
Comment (Biruntha) [inline README.md]: Put [Getting Started] (put link here) here.

File: sonarqube6/Package.md [hunk 3]
```diff
@@ -14,20 +19,27 @@ Using Ballerina SonarQube connector you can easily get important code quality me
...
 
 The following sections provide you with information on how to use the Ballerina SonarQube connector.
+>>>>>>> 9e35213a21b5901cb08b67cc32061d5417cfdcc5
 
-- [Getting started](#getting-started)
-- [Quick Testing](#quick-testing)
-- [Working with sonarqube connector actions](#working-with-sonarqube-connector-actions)
+| Ballerina Language Version                   | Connector Version           | API Version
+| ---------------------------------------------|:--------------------------:| :--------------:
+| 0.970.0-beta3                                | 0.8.7                      | 6.7.2
```
Comment (Biruntha) [inline sonarqube6/Package.md]: remove Connector Version here

File: README.md (hunk 2 above)
Comment (hasithajayasundara) [reply README.md]: Done

**Review by dimslee — APPROVED** [review]

**Pull request merged**